### PR TITLE
Ports Special Intents from Yog

### PR DESCRIPTION
**Pull request by Super3222**
[Changelogs]:

:cl: Super3222
fix: Fixes that annoying say bug where () shows up every time you talk.
/:cl:

### CODE REVIEW

**Comment by Super3222** [comment]
image of this working:

**Comment by Funce** [comment]
Title is a little Sketchy

File: code/game/say.dm [hunk 1]
```diff
@@ -71,7 +71,7 @@ GLOBAL_LIST_INIT(freqtospan, list(
 	return ""
 
 /atom/movable/proc/compose_job(atom/movable/speaker, message_langs, raw_message, radio_freq)
-	return "[" (" + speaker.GetJob() + ")"]"
+	return "[ speaker.GetJob() ? " (" +  speaker.GetJob() + ")" : ""]"
```
Comment (Funce) [inline code/game/say.dm]: I would suggest instead of calling the GetJob proc twice, to make it into a var to reference twice.

Comment (Super3222) [reply code/game/say.dm]: Sorry I have a single-line mentality

**Comment by Super3222** [comment]
I feel like this PR is broken, so I'm moving it to https://github.com/FTL13/FTL13/pull/2217.

**Pull request closed without merging**